pkgs/development/python-modules/pydantic/default.nix

```diff
@@ -1,138 +1,73 @@
-{ lib
-, stdenv
-, buildPythonPackage
-, autoflake
-, cython
-, devtools
-, email-validator
-, fetchFromGitHub
-, fetchpatch
-, pytest-mock
-, pytestCheckHook
-, python-dotenv
-, pythonAtLeast
-, pythonOlder
-, pyupgrade
-, typing-extensions
-# dependencies for building documentation.
-# docs fail to build in Darwin sandbox: https://github.com/samuelcolvin/pydantic/issues/4245
-, withDocs ? (stdenv.hostPlatform == stdenv.buildPlatform && !stdenv.isDarwin && pythonAtLeast "3.10")
-, ansi2html
-, markdown-include
-, mike
-, mkdocs
-, mkdocs-exclude
-, mkdocs-material
-, mdx-truly-sane-lists
-, sqlalchemy
-, ujson
-, orjson
-, hypothesis
-, libxcrypt
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  callPackage,
+  hatch-fancy-pypi-readme,
+  hatchling,
+  annotated-types,
+  pydantic-core,
+  typing-extensions,
+  email-validator,
+  pythonOlder,
+  pytestCheckHook,
+  dirty-equals,
+  pytest-examples,
+  pytest-mock,
+  faker,
 }:
```

Comment:
Can you keep the old formatting to make the diff smaller?

Reply:
Fixed in https://github.com/NixOS/nixpkgs/compare/0734be7a465c129a4ff92b1179ca1c8dcbf027a9..27b5cc26162f07368235823ea6f37df7627fd945